scripts/process_licenses.py

```diff
@@ -139,7 +139,8 @@ def main():
                     dependency_without_license_count += 1
                     missing_licenses_str += f"{dependency_lookup_key}: {license_name}\n"
                 else:
-                    raise Exception(f"The dependency {dependency_lookup_key} does not ship a license file, but neither is it not defined in ADDITIONAL_LICENSES_MAP")
+                    err_msg = f"The dependency {dependency_lookup_key} does not ship a license file, but neither is it not defined in ADDITIONAL_LICENSES_MAP"
```

Comment:
FYI: @sfc-gh-mbobowski 
I didn't know as well :D
https://docs.astral.sh/ruff/rules/f-string-in-exception/